earch('.responsive-listing-grid .v2-listing-card__info .text-body').each_with_index do |element, index|

```diff
@@ -0,0 +1,236 @@
+
```

Comment:
I think that there's some sort of file mismatch here, the file name doesn't make sense and you have a complete file of what seems to be a text editor documentation, could you double check this and remove?

Reply:
That piece of code is not meant to be permanent at all I wanted to doublecheck is Nokogiri was installed correctly, I'll delete that piece of code in a bit